snooty.toml

```
@@ -31,3 +31,4 @@ driver-short = "Node.js driver"
 mdb-server = "MongoDB Server"
 min-node-version = "v16.20.1"
 stable-api = "Stable API"
+language = "Javascript"
```

```suggestion
language = "JavaScript"
```

The S is always capitalized (but if we're mimicking a decision already made elsewhere feel free to ignore)

---

Thanks for the note!